CHANGELOG.md

```diff
@@ -1,7 +1,51 @@
 x.y.z Release notes (yyyy-MM-dd)
 =============================================================
 ### Enhancements
-* None.
+* Added initial support for geospatial queries on points.
+  There is no new dedicated type to store Geospatial points, instead points should
+  be stored as ([GeoJson-shaped](https://www.mongodb.com/docs/manual/reference/geojson/)) embedded object, as the example below.
+  ```swift
+  public class Location: EmbeddedObject {
```

Comment:
Why are we going for this approach instead of creating a dedicated wrapper class?

Reply:
This is the proposed approach at the moment, before we add a specific type for storing GeoJson data. Documentation for Swift is not yet added but this is what we are gonna suggest our users https://www.mongodb.com/docs/realm/sdk/dotnet/model-data/data-types/geospatials/#creating-a-geojson-compatible-class